community/xcb-util-xrm/APKBUILD

```diff
@@ -1,19 +1,29 @@
 # Maintainer: William Pitcock <nenolod@dereferenced.org>
 # Author: Jakub Skrzypnik <j.skrzypnik@openmailbox.org>
 pkgname=xcb-util-xrm
-pkgver=1.2
-pkgrel=1
+pkgver=1.3
+pkgrel=0
 pkgdesc="Utility functions for the X resource manager"
-url="https://github.com/Airblader/xcb-util-xrm/"
+url="https://github.com/Airblader/xcb-util-xrm"
 arch="all"
 license="MIT"
-makedepends="libx11-dev libxcb-dev xcb-util-dev util-macros bsd-compat-headers m4"
-source="https://github.com/Airblader/$pkgname/releases/download/v$pkgver/$pkgname-$pkgver.tar.gz"
+makedepends="autoconf automake bsd-compat-headers libtool libx11-dev libxcb-dev util-macros xcb-util-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/Airblader/$pkgname/archive/v$pkgver/v$pkgver.tar.gz
+	xcb_util_common.m4
+	ax_compare_version.m4
+	xcb_util_m4_with_include_path.m4	
+	"
 subpackages="$pkgname-dev"
 builddir="$srcdir/$pkgname-$pkgver"
 
+prepare() {
+	default_prepare
+	cp "$startdir"/*.m4 m4/
+}
+
 build() {
 	cd "$builddir"
+	./autogen.sh
```

Comment:
Our preference is to run `autogen` on `prepare()`. I know there are few APKBUILDs also doing it in `build()`, but AFAIK prepare is the proper place. Please move.